node/src/options.rs

```diff
@@ -0,0 +1,23 @@
+//! Node configuration options
+
+use std::net::SocketAddr;
+use std::path::PathBuf;
+
+/// Mintlayer node binary
+#[derive(clap::Parser, Debug)]
+#[structopt(name = "mintlayer-node")]
+pub struct Options {
+    /// Where to write logs
+    #[clap(long, name = "PATH")]
+    pub log_path: Option<PathBuf>,
+
+    /// Address to bind RPC to
+    #[clap(long, name = "ADDR", default_value = "127.0.0.1:3030")]
```

Comment:
Keep in mind that there are two bind addresses, rpc and p2p (hence, 1. I'm opposed to calling it addr as it should have the word bind in it, and 2. it should indicate what this is binding)

Reply:
Ah, I see. That's OK. Keep it caps then.